samples/layouts/stepper/stepper-title-position-and-orientation-example/index.html

```diff
@@ -0,0 +1,67 @@
+<html>
+
+<head>
+  <title>Stepper Title Position And Orientation Example</title>
+  <meta charset="UTF-8" />
+
+  <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+  <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" type="text/css" />
+</head>
+
+<body>
+  <div id="root">
+    <div class="container sample">
+      <div class="radio-groups">
+        <div class="radio-group">
+          <label>Title position</label>
+          <div class="radio-group-container">
+            <igc-radio-group id="titlePosition" alignment="horizontal">
+              <igc-radio value="top">Top</igc-radio>
+              <igc-radio value="bottom">Bottom</igc-radio>
+              <igc-radio value="start">Start</igc-radio>
+              <igc-radio value="end">End</igc-radio>
+              <igc-radio value="" checked>Default</igc-radio>
+            </igc-radio-group>
+          </div>
+        </div>
+        <div class="radio-group">
+          <label>Orientation</label>
+          <div class="radio-group-container">
+            <igc-radio-group id="orientation" alignment="horizontal">
+              <igc-radio value="horizontal" checked>Horizontal</igc-radio>
+              <igc-radio value="vertical">Vertical</igc-radio>
+            </igc-radio-group>
+          </div>
+        </div>
+      </div>
+      <igc-stepper id="stepper">
+        <igc-step>
+          <span slot="title">Order</span>
+          </br>
+          <igc-button onclick="stepper.next()">NEXT</igc-button>
+        </igc-step>
+        <igc-step>
+          <span slot="title">Payment</span>
+          </br>
+          <igc-button onclick="stepper.prev()">PREVIOUS</igc-button>
+          <igc-button onclick="stepper.next()">NEXT</igc-button>
+        </igc-step>
+        <igc-step>
+          <span slot="title">Confirmation</span>
+          </br>
+          <igc-button onclick="stepper.prev()">PREVIOUS</igc-button>
+          <igc-button onclick="stepper.reset()">RESET</igc-button>
+        </igc-step>
```

Comment:
Same as stepper-steptypes-example, see comment above.

Reply:
Please see the reply to the commoent above.